javascript/ql/src/semmle/javascript/security/dataflow/MethodNameInjection.qll

```diff
@@ -0,0 +1,148 @@
+/**
+ * Provides a taint tracking configuration for reasoning about method invocations
+ * with a user-controlled method name.
+ */
+
+import javascript
+import semmle.javascript.frameworks.Express
+
+module MethodNameInjection {
+  private import DataFlow::FlowLabel
+  
+  /**
+   * A data flow source for method name injection.
+   */
+  abstract class Source extends DataFlow::Node {
+    /**
+     * Gets the flow label relevant for this source.
+     */
+    DataFlow::FlowLabel getFlowLabel() {
```

Comment:
Is this used anywhere?

Reply:
Ah, good catch. It was supposed to be used in `isSource`.